tests/stavenote_tests.ts

```diff
@@ -392,21 +392,27 @@ function drawBasic(options: TestOptions, contextBuilder: ContextBuilder): void {
   ];
   expect(noteStructs.length * 2);
 
-  const colorDescendants = (parentItem: SVGElement, color: string) => () =>
-    parentItem.querySelectorAll('*').forEach((child) => {
-      child.setAttribute('fill', color);
-      child.setAttribute('stroke', color);
```

Comment:
I notice the original code sets the 'stroke' as well. Do we want to support stroke style in your PR? Or do you think fill is sufficient?

Reply:
I was thinking about agreeing on how to proceed with fill and then add others